src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/PrimitiveDataContract.cs

```diff
@@ -346,6 +346,9 @@ internal sealed class NullPrimitiveDataContract : PrimitiveDataContract
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
             Justification = "This warns because the call to Base has the type annotated with DynamicallyAccessedMembers so it warns" +
             "when looking into the methods of NullPrimitiveDataContract. Because this just represents null, we suppress.")]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2111",
```

Comment:
Do we still need to suppress IL2026 above?

Reply:
Yup, the IL2026 is about members of NullPrimitiveDataContract annotated with RequiresUnreferencedCode, and the IL2111 is about members annotated with DynamicallyAccessedMembers. I can tweak the wording to make that clear.